rest/config_test.go

```diff
@@ -1213,7 +1262,13 @@ func TestExpandEnv(t *testing.T) {
 				require.Equal(t, v, value, "Unexpected value set for environment variable %q", k)
 			}
 			// Check environment variable substitutions.
-			actualConfig := expandEnv(test.inputConfig)
+			actualConfig, err := expandEnv(test.inputConfig)
+			if test.expectedError != nil {
+				errs, ok := err.(*multierror.Error)
+				require.True(t, ok)
+				require.Equal(t, test.expectedError.Len(), errs.Len())
+				assert.Equal(t, test.expectedError, errs)
+			}
```

Comment:
We're not ensuring `err` is nil here.

```suggestion
			} else {
				assert.NoError(t, err)
			}
```

Reply:
Yeah, this is was nice catch. It's committed to latest revision.